charts/firecloudorch/values.yaml

```diff
@@ -85,4 +85,25 @@ prometheus:
   enabled: true
   initContainerImage: 'alpine:3.12.0'
   jmxJarRepo: 'https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent'
-  jmxJarVersion: '0.13.0'
\ No newline at end of file
+  jmxJarVersion: '0.13.0'
+
+elasticsearch:
+  transportProbe:
+    # elasticsearch.transportProbe.enabled -- Enables a custom liveness probe to account for ES transport client losing connection when elasticsearch restarts.
+    enabled: true
+    # elasticsearch.transportProbe.initContainerImage -- Docker image to use to install tools needed for custom probe
+    initContainerImage: 'alpine:3.12.0'
+    # elasticsearch.transportProbe.appHostname -- hostname used for querying orch's status
+    appHostname: 'localhost:8080'
+    # elasticsearch.transportProbe.elasticHostname -- hostname for querying the status of the Elasticsearch cluster
+    elasticHostname: 'elasticsearch5a-master-0.elasticsearch5a-master-headless:9200'
+    # elasticsearch.transportProbe.spec -- Spec for the custom liveness probe 
+    spec:
+      exec:
+        command:
+        - ./etc/probe/probe.sh
+      timeoutSeconds: 5
```

Comment:
I think timeoutSeconds might be ignored for exec probes on GKE < 1.20 (we're on 1.18)

https://github.com/kubernetes/kubernetes/pull/94115

It might be a good idea to set a timeout on the probe's curl commands (`-m <seconds>`), curl's default timeout is an hour.

Reply:
Ah didn't know timeouts where ignored on the exec probes. Will definitely set timeouts on the curls in that case.